src/screens/CreateCommunity/index.js

```diff
@@ -208,7 +218,7 @@ const CreateCommunity = () => {
                   autoCapitalize="none"
                   autoCorrect={false}
                   style={styles.input}
-                  autoFocus={true}
+                  autoFocus={false}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Setting `autoFocus` to `false` is appropriate here since we're manually focusing the input using the `useRef` hook and `useEffect`.
```javascript
autoFocus={false}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->